region/brinstar/blue/Blue Brinstar Boulder Room.json

```diff
@@ -775,6 +775,21 @@
         "This strat is easily bypassed by jumping through the door, or if the door can be opened."
       ]
     },
+    {
+      "link": [3, 1],
+      "name": "Bomb Boost Water Escape Into IBJ",
+      "requires": [
+        "canBombJumpWaterEscape",
+        "canJumpIntoIBJ",
+        "h_canUnderwaterCrouchJump",
```

Comment:
I never got it without the crouch, but if it's not required I will remove it. I was planning on using that helper everywhere you crouch in water. Would you like it renamed somehow to not feel like it's different from a normal crouch jump most of the time?

Reply:
tldr; `h_canUnderwaterCrouchJump` -> `h_canUnderwaterCrouchJumpWithFlashSuit`

Practically speaking the `canBombJumpWaterEscape` always does an underwater crouch jump since it uses a soft-unmorph on the rejump, but you can go out of your way to avoid it here.  The problem right here is that bombjumpwaterescapes have a varying quality to their crouchJumps and it can seem like the helper is referencing that.  Crouch jumps might be missing from other places where you bombJumpWaterEscape but not doing them makes the trick harder.

What the helper is trying to say is that a crouch or straight jump can be done in water with a flash suit without activating it.  Knowing that it is possible to do that is a tech and would be described in a `moveWithFlashSuit` tech making this note a reminder and not the main way to document the skill.  The role of this helper is to show that you need this technique more than applying new requirements to the strat and it is still useful in that way but not much different from being in a note.  Pointless point: you can do this without a straight jump by bombing into the wall to start your IBJ.

`h_canUnderwaterCrouchJumpWithFlashSuit` is ugly long but
- it only works in water physics
- only applies to straight jumps and a crouch jump is more generally useful
- has the specific purpose of avoiding `{noFlashSuit}`